src/main/java/core/basesyntax/service/impl/TransactionProcessService.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+
+public class TransactionProcessService implements TransactService {
+    private static final int OPERATION = 0;
+    private static final String COMMA = ",";
+    private OperationStrategy activitiesStrategy;
+
+    public TransactionProcessService(OperationStrategy activitiesStrategy) {
+        this.activitiesStrategy = activitiesStrategy;
+    }
+
+    @Override
+    public HashMap<String, Integer> process(String[] data) {
+        HashMap<String, Integer> database = new HashMap<>();
+        for (String datum : data) {
+            String[] check = datum.split(COMMA);
+            activitiesStrategy.get(check[OPERATION]).processCommand(database, check);
+        }
+        return database;
+    }
```

Comment:
you have no DB in your app
![image](https://github.com/mate-academy/jv-fruit-shop/assets/49767049/1a73aad3-7f37-4284-b33d-5cbc00496e59)
Here, you should work with this mock DB (it will be just a static final HashMap)